Update disks-copy-incremental-snapshot-across-regions.md (#124149)
Azure/azure-content: articles/virtual-machines/disks-copy-incremental-snapshot-across-regions.md

- If you use the REST API, you must use version 2020-12-01 or newer of the Azure Compute REST API.
- You can only copy one incremental snapshot of a particular disk at a time.
- Snapshots must be copied in the order they were created.

> - Copy of full snapshots to another region is not supported, only incremental snapshots are supported.

Reviewer: ```suggestion
- Only incremental snapshots can be copied across regions, full snapshots can't be copied across regions.
```

> - Only incremental snapshots can be copied across regions, full snapshots can't be copied across regions.

Reviewer 2: ```suggestion
- Only incremental snapshots can be copied across regions. Full snapshots can't be copied across regions.
```
Minor edit
